stubs/entrypoints/METADATA.toml

```diff
@@ -0,0 +1,2 @@
+version = "0.1"
```

Comment:
Could you set this to the latest version instead?

Reply:
@JelleZijlstra done, bumped to `0.3` which is currently the last release on PyPI.